apps/dashboard/app/views/batch_connect/shared/_full_page_spinner.html.erb

```diff
@@ -0,0 +1,15 @@
+<script nonce="<%= content_security_policy_nonce %>">
+jQuery(function (){
```

Comment:
Can we just add the javascript file instead? outside of security reasons - we kinda just want to stop inlining javascript just so we don't have to recall where things are coming from.

Reply:
will do

Reply:
fixed